src/openms/include/OpenMS/FORMAT/XQuestResultXMLFile.h

```diff
@@ -48,11 +48,11 @@ namespace OpenMS
     XQuestResultXMLFile();
     ~XQuestResultXMLFile();
 
-    void load(const String &,  // filename
-              std::vector< std::vector< PeptideIdentification > > &, // Vector of encompassed spectra, each containing associated PeptideIdentifications
-              std::vector< ProteinIdentification > &,  // The protein identification of the xQuest result file
-              Size = 0,  // Minimum number of PeptideIdentification a spectrum must contain to be loaded
-              bool = false); // Whether the OpenXQuest data will be loaded as Meta Values also into the peptide hits, instead just into the PeptideIdentification
+    void load(const String & filename,  // filename
```

Comment:
the documentation should be in doxygen format, not inside the declaration please! use
```
/* @brief
 *
 * @param filename ...
 * @param csms ...
 */
```